configs/envoy_service_to_service.template.json

```diff
@@ -92,7 +92,7 @@
   "listeners": [
   {{ ingress_listener(9211) }},
```

Comment:
this should be tcp://0.0.0.0:9211. Can you see how this is passing tests? I would think it should fail due to invalid URL?